experimental/framework/tree-react-api/src/test/tsconfig.json

```diff
@@ -1,20 +1,15 @@
 {
-	"extends": "@fluidframework/build-common/ts-common-config.json",
+	"extends": "../../../../../common/build/build-common/tsconfig.test.node16.json",
 	"compilerOptions": {
-		"lib": ["ES2019", "ES2018.Promise", "DOM", "DOM.Iterable"],
-		"noImplicitAny": true,
-		"noUnusedLocals": false,
-		"noImplicitOverride": true,
 		"rootDir": "./",
 		"outDir": "../../lib/test",
-		"module": "Node16",
-		"moduleResolution": "Node16",
-		"types": ["node", "mocha"],
+		"types": ["mocha", "node"],
+		"noUnusedLocals": false,
```

Comment:
Consider adding a comment to this explaining why it's overridden.

Reply:
removed